### PR TITLE
SPLICE-1702 Removing LocatedRow From Execution Stack

### PR DESCRIPTION
**Pull request by jleach4**
LocatedRow was a miss from the beginning.  Finally got around on a flight to taking it out.

### CODE REVIEW

**Comment by snowspeeder** [comment]
can you please fix one conflict

File: hbase_sql/src/main/java/com/splicemachine/derby/stream/spark/SparkDataSet.java [hunk 1]
```diff
@@ -508,11 +505,11 @@ public void checkOutputSpecs(JobContext job) throws FileAlreadyExistsException,
                 fileOut = new GZIPOutputStream(fileOut);
             }
             final ExportExecRowWriter rowWriter = ExportFunction.initializeRowWriter(fileOut, op.getExportParams());
-            return new RecordWriter<Void, LocatedRow>() {
+            return new RecordWriter<Void, ExecRow>() {
                 @Override
-                public void write(Void _, LocatedRow locatedRow) throws IOException, InterruptedException {
+                public void write(Void _, ExecRow locatedRow) throws IOException, InterruptedException {
                     try {
```
Comment (jyuanca) [inline hbase_sql/src/main/java/com/splicemachine/derby/stream/spark/SparkDataSet.java]: Rename variable name to valueRow?

File: hbase_sql/src/main/java/com/splicemachine/example/ExecRowToVectorFunction.java [hunk 2]
```diff
@@ -47,7 +46,7 @@ public void readExternal(ObjectInput in) throws IOException, ClassNotFoundExcept
     }
 
     @Override
-    public Vector call(LocatedRow locatedRow) throws Exception {
-        return SparkMLibUtils.convertExecRowToVector(locatedRow.getRow(),fieldsToConvert);
+    public Vector call(ExecRow locatedRow) throws Exception {
+        return SparkMLibUtils.convertExecRowToVector(locatedRow,fieldsToConvert);
```
Comment (jyuanca) [inline hbase_sql/src/main/java/com/splicemachine/example/ExecRowToVectorFunction.java]: ditto

File: splice_machine/src/main/java/com/splicemachine/derby/impl/sql/execute/operations/SpliceBaseOperation.java [hunk 3]
```diff
@@ -72,7 +72,7 @@
     protected boolean isOpen=true;
     protected int resultSetNumber;
     protected OperationInformation operationInformation;
-    protected LocatedRow locatedRow;
+    protected ExecRow locatedRow;
     protected StatementContext statementContext;
```
Comment (jyuanca) [inline splice_machine/src/main/java/com/splicemachine/derby/impl/sql/execute/operations/SpliceBaseOperation.java]: ditto

File: hbase_sql/src/main/resources/core-site.xml [hunk 4]
```diff
@@ -36,7 +36,12 @@
 
     <property>
         <name>fs.defaultFS</name>
-        <value>file:///</value>
```
Comment (dgomezferro) [inline hbase_sql/src/main/resources/core-site.xml]: I would split the filesystem change into a different issue, it has nothing to do with LocatedRow.

**Comment by dgomezferro** [comment]
I would try to get this merged just after the next sprint build, just in case it breaks something. We should try to merge it ASAP otherwise it will keep accumulating conflicts. 

I think we should also merge it to 2.5, otherwise it will be a pain to backport stuff between master and 2.5

**Comment by snowspeeder** [comment]
latest SPLICE-1702-v3 branch has been merged, closing this out.

**Pull request closed without merging**